### PR TITLE
Update collection-details.md

### PR DESCRIPTION
**Pull request by deryq**
Removed providers.

### CODE REVIEW

File: _docs/collection-details.md [hunk 1]
```diff
@@ -32,7 +31,7 @@ To initiate a GHS mobile collections please use the following details:
 
 Once the transaction is created the specified phone number will receive instructions on how to pay in the requested amount.
 
-Please note that Interpay GHS Mobile collections does not work with Vodafone Cash. Also if the customer does not have enough funds to pay the transaction they might not receive the payment prompt to their phone. In this case the transaction request should be re-sent, once the customer has confirmed they have enough funds.
+Please note that GHS Mobile collections does not work with Vodafone Cash. Also if the customer does not have enough funds to pay the transaction they might not receive the payment prompt to their phone. In this case the transaction request should be re-sent, once the customer has confirmed they have enough funds.
```
Comment (sztupy) [inline _docs/collection-details.md:34]: Is this still true?

Comment (sztupy) [reply _docs/collection-details.md:34]: (Vodafone not working). I believe this was fixed already by the provider

File: _docs/collection-details.md [hunk 2]
```diff
@@ -110,7 +108,6 @@ To initiate EUR IBAN collections please use the following details:
 "payin_methods": [
   {
     "type": "EUR::Bank",
```
Comment (sztupy) [inline _docs/collection-details.md:110]: you also have to remove the leading comma from the line above

File: _docs/collection-details.md [hunk 3]
```diff
@@ -155,7 +152,6 @@ To initiate GBP Faster Payments collections please use the following details:
 "payin_methods": [
   {
     "type": "GBP::Bank", // or EUR::Bank
```
Comment (sztupy) [inline _docs/collection-details.md:154]: you also have to remove the leading comma from the line above

File: _docs/collection-details.md (hunk 2 above)
Comment (jara) [inline _docs/collection-details.md:110]:
```suggestion
    "type": "EUR::Bank"
```

File: _docs/collection-details.md (hunk 3 above)
Comment (jara) [inline _docs/collection-details.md:154]:
```suggestion
    "type": "GBP::Bank" // or EUR::Bank
```

**Comment by deryq** [comment]
Merged in another PR

**Pull request closed without merging**